<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Welcome</class>
 <widget class="QDialog" name="Welcome">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>653</width>
    <height>490</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Welcome</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="horizontalSpacing">
    <number>8</number>
   </property>
   <item row="6" column="0">
    <widget class="QCheckBox" name="defaultSettings">
     <property name="text">
      <string>Use Default Settings</string>
     </property>
    </widget>
   </item>
   <item row="11" column="2">
    <widget class="QPushButton" name="startVideo">
     <property name="text">
      <string>Play video</string>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QSlider" name="kernelSize">
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>99</number>
     </property>
     <property name="singleStep">
      <number>1</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="6" column="2">
    <widget class="QPushButton" name="cameraOptions">
     <property name="text">
      <string>Camera Options</string>
     </property>
    </widget>
   </item>
   <item row="4" column="3">
    <widget class="QLabel" name="labelDrawTh">
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QSlider" name="bufferSize">
     <property name="minimum">
      <number>2</number>
     </property>
     <property name="maximum">
      <number>40</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="0" column="3">
    <widget class="QLabel" name="Blabel">
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>Adaptive treshold shift</string>
     </property>
    </widget>
   </item>
   <item row="14" column="2">
    <widget class="QCheckBox" name="originalVideo">
     <property name="text">
      <string>Original video</string>
     </property>
    </widget>
   </item>
   <item row="7" column="2" alignment="Qt::AlignLeft">
    <widget class="QSpinBox" name="device"/>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Size of Kernel</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Hand identification threshold</string>
     </property>
    </widget>
   </item>
   <item row="19" column="0">
    <widget class="QPushButton" name="saveSketch">
     <property name="text">
      <string>Save Sketch</string>
     </property>
    </widget>
   </item>
   <item row="15" column="0">
    <widget class="QCheckBox" name="difference">
     <property name="text">
      <string>Show difference image</string>
     </property>
    </widget>
   </item>
   <item row="15" column="2">
    <widget class="QCheckBox" name="roiHand">
     <property name="text">
      <string>Show ROI of the hand</string>
     </property>
    </widget>
   </item>
   <item row="12" column="2">
    <widget class="QPushButton" name="Start">
     <property name="text">
      <string>Start Drawing</string>
     </property>
    </widget>
   </item>
   <item row="1" column="3">
    <widget class="QLabel" name="KLabel">
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
   </item>
   <item row="7" column="0">
    <widget class="QPushButton" name="Videopreview">
     <property name="text">
      <string>Video Preview</string>
     </property>
    </widget>
   </item>
   <item row="3" column="3">
    <widget class="QLabel" name="HLabel">
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
   </item>
   <item row="13" column="2">
    <widget class="QPushButton" name="finishDraw">
     <property name="text">
      <string>Finish</string>
     </property>
    </widget>
   </item>
   <item row="10" column="2">
    <widget class="QLabel" name="fileName">
     <property name="text">
      <string>No file loaded</string>
     </property>
    </widget>
   </item>
   <item row="14" column="0">
    <widget class="QPushButton" name="imageCl">
     <property name="text">
      <string>Clean image</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Size of Buffer</string>
     </property>
    </widget>
   </item>
   <item row="10" column="0">
    <widget class="QPushButton" name="loadFile">
     <property name="text">
      <string>Load File</string>
     </property>
    </widget>
   </item>
   <item row="3" column="2">
    <widget class="QSlider" name="handIdentThr">
     <property name="minimum">
      <number>25</number>
     </property>
     <property name="maximum">
      <number>254</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="4" column="2">
    <widget class="QSlider" name="drawTh">
     <property name="minimum">
      <number>-10</number>
     </property>
     <property name="maximum">
      <number>30</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="20" column="0">
    <widget class="QPushButton" name="filtering">
     <property name="text">
      <string>Filter application</string>
     </property>
    </widget>
   </item>
   <item row="20" column="2">
    <widget class="QPushButton" name="Exit">
     <property name="text">
      <string>Exit</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>Start</sender>
   <signal>clicked()</signal>
   <receiver>Start</receiver>
   <slot>showMenu()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>516</x>
     <y>348</y>
    </hint>
    <hint type="destinationlabel">
     <x>516</x>
     <y>348</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
